DOC Recommend installation of spamprotection (#966)
silverstripe/silverstripe-userforms: readme.md

## Installation

```sh
composer require silverstripe/userforms
```

You'll also need to run `dev/build`. You should see a new page type in the CMS 'User Defined Form'. This has a new 'Form' tab which has your form builder.

## SPAM protection

> This module does not include SPAM protection out of the box. Consider installing `silverstripe/spamprotection` as well. This will allow CMS users to add a "SPAM Protection Field" to user forms.

Reviewer: Need to note in this readme that `silverstripe/spamprotection` is just an abstraction, you also need to add another module on top of that.  As mentioned in the parent issue, installing this module without the implementation will give a false sense of security as users can add a spam protection field that does nothing. There are some common modules list in the readme of `silverstripe/spamprotection`

Most commonly used one I've seen by far is the undefinedoffset/nocaptcha (google recaptcha), though as mentioned on the parent issue of this PR, we don't officially support that module, and what's most popular may change, so we can't really recommend use of that module within this readme.

> ## SPAM protection
> 
> This module does not include SPAM protection out of the box. Consider installing `silverstripe/spamprotection` as well. This will allow CMS users to add a "SPAM Protection Field" to user forms.
> 
> ```sh
> composer require silverstripe/spamprotection
> ```
> 
> Read the [SpamProtection Module README](https://github.com/silverstripe/silverstripe-spamprotection/) for details on how to configure this module.
> 

Reviewer 2: Agree with @emteknetnz's point. I've kind of butchered your write up, but here's a suggestion.

```suggestion
## Spam protection

This module does not include spam protection out of the box. Without it, it's likely that your submissions could contain a considerable amount of spam. For public facing forms it is encouraged you review and install the following module plus one of the recommended 'verification system' modules outlined in the README.

Read the [SpamProtection Module README](https://github.com/silverstripe/silverstripe-spamprotection/) for details on how to configure this module.

```

You'll also need to run `dev/build`. You should see a new page type in the CMS 'User Defined Form'. This has a new 'Form' tab which has your form builder.

## SPAM protection

This module does not include SPAM protection out of the box. Consider installing `silverstripe/spamprotection` as well. This will allow CMS users to add a "SPAM Protection Field" to user forms.

```sh

> $ composer require silverstripe/userforms

Author: ```suggestion
composer require silverstripe/spamprotection
```

> 
> ```sh
> $ composer require silverstripe/userforms
> ```

Reviewer 2: ```suggestion
```
Is this line supposed to be here?

Author: Oups ... copy-pasted the line above and forgot to change the module name.

Author: Fixed
